util/network-devp2p/src/math.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Coinbase, Inc.
```

Comment:
I'm not sure if we can accept this license header, cc @5chdn

Reply:
it's definitely not compatible with the CLA

Reply:
I'm willing to change it if necessary, but my reading of the [CLA](https://www.parity.io/cla.html) made me think it's OK. Section 2.1 a of the CLA says "You retain ownership of the Copyright and Patent Claims in your Contributions and have the same rights to use or license the Contributions which You would have had without entering into the Agreement."

Reply:
If you don't mind, let's share the copyright header. This will remove the burdon of future contributors or core developers to keep it updated. What do you think?

    // Copyright 2018 Coinbase Inc., Parity Technologies (UK) Ltd.

Reply:
Updated